pkg/metrics/metrics.go

```diff
@@ -159,6 +159,108 @@ var (
 		Name:      "policy_l7_received_total",
 		Help:      "Number of total L7 received requests/responses",
 	})
+
+	// Status
+
+	// ControllersFailing is a count of failing controllers
+	ControllersFailing = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "controllers_failing_total",
+		Help:      "Number of faling controllers",
+	})
+
+	// IPv4Addresses is a count of allocated IPv4 addresses in the cluster
+	IPv4Addresses = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "ipv4_addresses_total",
+		Help:      "Number of allocated IPv4 addresses",
+	})
+
+	// IPv6Addresses is a count of allocated IPv6 addresses in the cluster
+	IPv6Addresses = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "ipv6_addresses_total",
+		Help:      "Number of allocated IPv6 addresses",
+	})
+
+	// UnreachableNodes is a count of nodes which have no ICMP or HTTP L3
+	// connectivity
+	UnreachableNodes = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "unreachable_nodes_total",
+		Help:      "Number of nodes that cannot be reached",
+	})
+
+	// UnreachableHealthEndpoints is a count of node endpoints which have no
+	// ICMP or HTTP L3 connectivity
+	UnreachableHealthEndpoints = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "unreachable_health_endpoints_total",
+		Help:      "Number of health endpoints",
+	})
+
+	// AvgNodesICMPLatency is an average ICMP latency to nodes in the
+	// cluster
+	AvgNodesICMPLatency = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "avg_nodes_icmp_latency",
```

Comment:
You'll want to explicitly name a unit in the metric name, like `http_request_duration_seconds` from the Prometheus client library.